extension/popup/popup.html

```diff
@@ -37,16 +37,30 @@
             </div>
             <div id="most-recent-query">Search for shoes on Amazon</div>
           </div>
-          <div id="moz-voice-privacy">
-            <a href="">How Mozilla ensures voice privacy</a>
-          </div>
         </div>
         <div id="text-input">
           <span id="text-input-field" contenteditable="true">&nbsp;</span>
           <div id="send-btn-wrapper">
               <button id="send-text-input">GO</button>
           </div>
         </div>
+        <div id="card">
+            <div id="card-header"></div>
+            <div id="card-image"><img src=""/></div>
+            <div id="card-summary"></div>
+            <div id="card-ack">
+              <div id="ddg-ack">
+                <img id="ddg-logo" src="./images/dax-logo.svg"/><a href="https://duckduckgo.com/">Results from DuckDuckGo</a>
+              </div>
+              <div class="sep"></div>
+              <div id="source-ack">
+                <span>Source:</span>
+                <div id="card-source">
+                  <a id="card-source-link"></a>
```

Comment:
Do we need a `target=_blank` on this? I'm honestly not sure, I can't remember what the link behavior is here.

Reply:
Oh, good call. I hadn't tested that. It looks like by default, and with target="_blank", it opens in a new window, even though my browser preference is to open links in new tabs. I imagine our ideal behavior is new tab within the same window, right?

Reply:
Ideally, yes... I vaguely recall using `browser.tabs.create()` to do this in the past because it's the only way I could get it to work in a new tab.

Reply:
Ok, changed this to use `browser.tabs.create()`. Listening for clicks on that link also means we can dismiss the popup after navigating to that new tab, which is handy.